src/components/breadcrumbs/examples/breadcrumbs-icon-only.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Using icons
+ * For an improved accessibility, you are required to
+ * provide a `text` for each item in the breadcrumbs.
+ * But each item can have an optional icon too.
+ *
+ * However, in some UIs, the design might require
+ * hiding the text and relying on an icon to visualize
+ * an item in the path.
+ *
+ * In this case you can set the `iconOnly` to
+ * `true` on the desired items.
+ *
+ * :::note
+ * The last item (current step) will always
+ * display both an icon and the text.
+ * :::
+ */
+@Component({
+    tag: 'limel-example-breadcrumbs-icons',
+    shadow: true,
+})
+export class BreadcrumbsExampleIcons {
+    @State()
+    private items = [
+        {
+            text: 'Home',
+            icon: 'smart_home',
+            iconOnly: true,
+        },
+        {
+            text: 'Products',
+            icon: 'shop',
+        },
+        {
+            text: 'Phones',
+            icon: 'google_mobile_copyrighted',
+        },
+        {
+            text: 'Accessories',
```

Comment:
In the note it say "The last item (current step) will always display both an icon and the text." but there is no icon in the last item.
<img width="1129" alt="Screenshot 2023-09-20 at 4 03 04 PM" src="https://github.com/Lundalogik/lime-elements/assets/22968497/aa2b1cc2-640e-4695-8664-2f5a81e71d5a">

Reply:
Good catch! Will fixup